crates/chia-protocol/src/pos_quality.rs

```diff
@@ -4,7 +4,9 @@
 pub const UI_ACTUAL_SPACE_CONSTANT_FACTOR: f32 = 0.78;
 
 // TODO: Update this when new plot format releases
-pub fn expected_plot_size(k: u32) -> u64 {
+#[cfg(feature = "py-bindings")]
+#[pyo3::pyfunction]
```

Comment:
I think you have to do `#[cfg_attr(feature = "py-bindings", pyo3::pyfunction)]` instead?

Reply:
@matt-o-how what happened with this? Did you try it and it didn't work?

Reply:
Correct - I tried this change and reverted it